opentok/opentok.py

```diff
@@ -90,14 +90,16 @@ def __init__(
         api_secret,
         api_url="https://api.opentok.com",
         timeout=None,
+        app_name=None,
         app_version=None,
     ):
         self.api_key = str(api_key)
         self.api_secret = api_secret
         self.timeout = timeout
         self._proxies = None
         self.endpoints = Endpoints(api_url, self.api_key)
-        self._app_version = __version__ if app_version == None else app_version
+        self._app_version = app_version
```

Comment:
This should be reverted so that we keep the original behavior (see other suggested edits)

Reply:
@dragonmantank This change was added in order to handle a custom app name and version (provided by the user). That's why line # 100 was removed because I was using the version (THE SDK VERSION) here. Lines # 101 and 102 were added to handle this custom information